src/Application/Cli/PreviewCommand.php

```diff
@@ -78,10 +77,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (!$this->isFound($input->getOption('livereload'))) {
-            $output->writeln('<error>Impossible to launch Livereload, did you forgot to install it with "pip install livereload" (sudo maybe required)?</error>');
+        if ($input->getOption('livereload')) {
+            if ($input->getOption('livereload') === null) {
+                $input->setOption('livereload', 'livereload');
+            }
```

Comment:
Based on a few manual tests, if `livereload` option is not set, the value is `false` and not `null` so should handle all use cases you mentioned earlier.